punnetsquaremaker.py

```diff
@@ -15,7 +15,28 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 # The author can be reached by email at catherine.moresco@gmail.com.
+# Improved with Latex tables by Remyslab, email at remyslab@laposte.net.
 
+import datetime
+
+def headtex():
+	f= open(name, 'w')
```

Comment:
nit: please use spaces on either side of equals signs (here and in the rest of the files)

https://google.github.io/styleguide/pyguide.html?showone=Whitespace#Whitespace